news/2024/2024-04-06-new-featured-artists-grabbitz-oddeeo-pas-lam-system-and-reaper.md

```diff
@@ -0,0 +1,75 @@
+
+If you're not into any of these maps, [download the beatmap template here](https://assets.ppy.sh/artists/410/Songs/Grabbitz%20-%20Way%20Too%20Deep.osz) and make your own!
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/artists/410/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### ODDEEO + Karma Wears White Ties - Chinatown Blues \[feat. Megpoid Gumi\]
+
+*Chinatown Blues*, the one and only retro pop song combining VOCALOID and human vocals by [**ODDEEO**](https://osu.ppy.sh/beatmaps/artists/411) and *Karma Wears White Ties*, made its way to osu! thanks to [Cloudchaser](https://osu.ppy.sh/users/1576498)'s [equally retro-outfitted beatmap](https://osu.ppy.sh/beatmapsets/883669).
+
+Try that for yourself, or [design your own map with this template](https://assets.ppy.sh/artists/411/Songs/ODDEEO%20%2B%20Karma%20Wears%20White%20Ties%20-%20Chinatown%20Blues%20%5Bfeat.%20Megpoid%20Gumi%5D.osz)!
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/artists/411/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### Pa's Lam System - City Lights Feat. EVO+, Jinmenusagi (Pa's Lam System Remix)
+
+[**Pa's Lam System**](https://osu.ppy.sh/beatmaps/artists/412)'s future bass remix of *City Lights* was an early pillar of the emerging tech craze via [this Loved map](https://osu.ppy.sh/beatmapsets/575809) by [hehe](https://osu.ppy.sh/users/2123087) and [byfar](https://osu.ppy.sh/users/4674054)! A few years later, [Kukkai](https://osu.ppy.sh/users/7811952) contributed to this song's popularity with [a full osu!catch spread](https://osu.ppy.sh/beatmapsets/1486745)!
+
+[Here's a beatmap template.](https://assets.ppy.sh/artists/412/Songs/Pa_s%20Lam%20System%20-%20City%20Lights%20Feat.%20EVO%2B%20%2C%20Jinmenusagi%20(Pa_s%20Lam%20System%20Remix).osz) You know what to do.
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/artists/412/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### REAPER - HEATSEEKER
+
+[**REAPER**](https://osu.ppy.sh/beatmaps/artists/413) REALLY LIKES TO SCREAM. THAT'S ALSO WHAT YOU'LL BE DOING WHILE PLAYING [THIS MAP OF **HEATSEEKER**](https://osu.ppy.sh/beatmapsets/1739056) HOSTED BY... [keevy](https://osu.ppy.sh/users/10584295).
+
+Alternatively, you can relax by [making your own map with this beatmap template](https://assets.ppy.sh/artists/413/Songs/REAPER%20-%20HEATSEEKER.osz).
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/artists/413/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+---
+
+As fun as it may be to announce 4 artists at once, we'll be jumping back to rea1ty next weekend with just one new Featured Artist!
```

Comment:
```suggestion
As fun as it may be to announce 4 artists at once, we'll be jumping back to reality next weekend with just one new Featured Artist!
```

Reply:
how did i